ci/jenkins/templates/utils/macros.j2

```diff
@@ -33,7 +33,7 @@ sh(
 {% set test_dir_name = name.replace(":", "").replace(" ", "-").replace("-", "_")|string %}
 def {{ method_name }}(node_type='{{ node }}-SPOT', on_demand=false) {
   if ({{ condition }}) {
-    if (on_demand==true) {
+    if (on_demand==true or 'ARM' in node_type) {
```

Comment:
without being wrapped in `{{ ... }}` this happens inside groovy at runtime, so `in` and `or` don't work
```suggestion
    if (on_demand==true || node_type.contains('ARM')) {
```